app/eq.py

```diff
@@ -87,31 +98,33 @@ async def build(self):
             self._language_code = self._sample_attributes['language']
 
         self._payload = {
-            'jti': str(uuid4()),  # required by eQ for creating a new claim
-            'tx_id': self.
-            _tx_id,  # not required by eQ (will generate if does not exist)
+            'jti': str(uuid4()),
+            'tx_id': self._tx_id,
             'iat': int(time.time()),
-            'exp': int(time.time() +
-                       (5 * 60)),  # required by eQ for creating a new claim
-            'collection_exercise_sid': self._collex_id,  # required by eQ
+            'exp': int(time.time() + (5 * 60)),
+            'collection_exercise_sid': self._collex_id,
             'region_code': self.convert_region_code(self._region),
-            'ru_ref': self._uprn,  # new payload requires uprn to be ru_ref
-            'case_id':
-            self._case_id,  # not required by eQ but useful for downstream
+            # 'ru_ref': self._uprn,
+            'ru_ref': self._questionnaire_id,  # SOCINT-258 - temporary for use with POC
+            'case_id': self._case_id,
             'language_code': self._language_code,
-            'display_address':
-            self.build_display_address(self._sample_attributes),
+            'display_address': self.build_display_address(self._sample_attributes),
             'response_id': self._response_id,
             'account_service_url': self._account_service_url,
-            'account_service_log_out_url':
-            self._account_service_log_out_url,  # required for save/continue
+            'account_service_log_out_url': self._account_service_log_out_url,
             'channel': self._channel,
```

Comment:
JIRA asks for "RH" but we are providing "rh".  Is that OK ? Needs confirmation.

Reply:
Already discussed with Paul and Sam. It needs to be 'rh'.

Reply:
OK